gpt_index/readers/chroma.py

```diff
@@ -2,8 +2,8 @@
 
 from typing import Any, List, Optional, Union
 
-from gpt_index.readers.base import BaseReader
-from gpt_index.readers.schema.base import Document
+from llama_index.readers.base import BaseReader
```

Comment:
let's keep imports as gpt_index (we have an automated script that creates the llama_index package)

Reply:
🙏